config/overlays/odh/kustomization.yaml

```diff
@@ -23,6 +23,38 @@ replacements:
       name: kserve-controller-manager
     fieldPaths:
     - spec.template.spec.containers.[name=manager].image
+- source:
+    kind: ConfigMap
+    name: kserve-parameters
+    fieldPath: data.kserve-storage-initializer
```

Comment:
maybe this should be `data.kserve-storage-initializer.image` or `data.kserve-storage-initializer["image"]` (not sure how kustomize treats nesting in json the same as yaml or not) 
If it works it should be the same solution for the rest as well

Reply:
Tried this, unfortunately .image results in an error:
```
Error: error looking up replacement source: wrong node kind: expected MappingNode but got ScalarNode: node contents:
quay.io/opendatahub/kserve-storage-initializer:latest
```

and ["image"] results in: 
```
Error: fieldPath data.kserve-storage-initializer["image"] is missing for replacement source ConfigMap.[noVer].[noGrp]/kserve-parameters.[noNs]
 ```